core/src/model/LivingEntity.java

```diff
@@ -0,0 +1,161 @@
+package model;
+
+import com.dongbat.jbump.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Parent class for all entities that have movement and health
+ */
+public abstract class LivingEntity extends Entity {
+    private float speed;
+    private boolean inMotion = false;
+    private float maxHealth;
+    private float currentHealth;
+    private float collisionDamage;
```

Comment:
Maybe keep it as only "damage" becasue the damage number should be the defualt one while things like weapons damage amplify/gets added onto this damage.